src/dr/app/beauti/options/BeautiOptions.java

```diff
@@ -254,7 +254,11 @@ public List<Parameter> selectParameters(List<Parameter> parameters) {
             Set<PartitionSubstitutionModel> substitutionModels = new LinkedHashSet<>();
             for (AbstractPartitionData partition : getDataPartitions()) {
                 if (partition.getPartitionClockModel() == model) {
-                    substitutionModels.add(partition.getPartitionSubstitutionModel());
+                    PartitionSubstitutionModel substitutionModel = partition.getPartitionSubstitutionModel();
```

Comment:
DataType.Tree partitions don't have substitution models. This ensures we don't add 'null' substitution models t